src/vaults/mainnet.json

```diff
@@ -0,0 +1,214 @@
+    },
+    {
+      "beraRewardsVault": "0x315D38ee321608cB8520D563750D64aC1BB17344",
+      "slug": "kodiak-usdc.e-honey",
+      "stakeTokenAddress": "0xb73deE52F38539bA854979eab6342A60dD4C8c03"
+    },
+    {
+      "beraRewardsVault": "0x75Da1D58A8d97f7b626224E7E71fa44DB13689f2",
+      "slug": "kodiak-usde-honey",
+      "stakeTokenAddress": "0xE5A2ab5D2fb268E5fF43A5564e44c3309609aFF9"
+    },
+    {
+      "beraRewardsVault": "0x524fc4f013d66e9fed424e58c5ada7d5b139761d",
+      "slug": "kodiak-wbera-bitcoin",
+      "stakeTokenAddress": "0x2cafe99db26cf86ae8587f5934830d25ad5c3cb3"
+    },
+    {
+      "beraRewardsVault": "0x45325df4a6a6ebd268f4693474aaaa1f3f0ce8ca",
+      "slug": "kodiak-wbera-honey",
+      "stakeTokenAddress": "0x4a254B11810B8EBb63C5468E438FC561Cb1bB1da"
+    },
+    {
+      "beraRewardsVault": "0xa2c5adB20A446Fa71A1762002E3C9B4Dd37DBAf4",
+      "slug": "kodiak-wbera-ibera",
+      "stakeTokenAddress": "0xe3eeb9e48934634d8b5b39a0d15dd89ee0f969c4"
+    },
+    {
+      "beraRewardsVault": "0x3Be1bE98eFAcA8c1Eb786Cbf38234c84B5052EeB",
+      "slug": "kodiak-wbera-ibgt",
+      "stakeTokenAddress": "0x564f011d557aad1ca09bfc956eb8a17c35d490e0"
+    },
+    {
+      "beraRewardsVault": "0x6b23e121c6a13c959d69493ea5ca015a5847596a",
+      "slug": "kodiak-wbera-ramen",
+      "stakeTokenAddress": "0x93a913351cae2d8c82c4b85f699726947eb76d32"
+    },
+    {
+      "beraRewardsVault": "0x9Df027Ed06635ecDfe811009f2291d347eA90357",
+      "slug": "kodiak-wbtc-honey",
+      "stakeTokenAddress": "0xf6b16E73d3b0e2784AAe8C4cd06099BE65d092Bf"
+    },
+    {
+      "beraRewardsVault": "0x10545e882f2ac24f0450fed9758d9c7aeaeb2f99",
+      "slug": "kodiak-wbtc-solvbtc",
+      "stakeTokenAddress": "0x3879451f4f69f0c2d37cad45319cff2e7d29c596"
+    },
+    {
+      "beraRewardsVault": "0xeec2ad7bb37374229860265e644f4e2693b23fdd",
+      "slug": "kodiak-wbtc-wbera",
+      "stakeTokenAddress": "0xf06ea29fcf4765200742d29e685973a1870eac98"
+    },
+    {
+      "beraRewardsVault": "0x0308f87226efaCC51983bEa842Be57Dc45dE772c",
+      "slug": "kodiak-wbtc-weth",
+      "stakeTokenAddress": "0x58FDB6EEbf7df7Ce4137994436fb0e629Bb84b84"
+    },
+    {
+      "beraRewardsVault": "0xedD27cE3793d655E3d15E29e297F4848De1EF092",
+      "slug": "kodiak-weth-beraeth",
+      "stakeTokenAddress": "0x03bCcF796cDef61064c4a2EffdD21f1AC8C29E92"
+    },
+    {
+      "beraRewardsVault": "0xC6506D205C92F9Acd69779eD0C547557F3AD4607",
+      "slug": "kodiak-weth-honey",
+      "stakeTokenAddress": "0xf6c6Be0FF6d6F70A04dBE4F1aDE62cB23053Bd95"
+    },
+    {
+      "beraRewardsVault": "0xcD1982B07ADFa787a88Dc1157b9fa3221d25fcaF",
+      "slug": "kodiak-weth-stone",
+      "stakeTokenAddress": "0x97431F104be73FC0e6fc731cE84486DA05C48871"
+    },
+    {
+      "beraRewardsVault": "0xfb657cd154e661ddf3e229529d92545640b19292",
+      "slug": "kodiak-weth-wbera",
+      "stakeTokenAddress": "0x9659dc8c1565E0bd82627267e3b4eEd1a377ebE6"
+    }
+  ]
```

Comment:
Missing YEET-WBERA (0x0710abffb1a54211a5e88d18bf9854cba86d0819) vault.
Add it please as well as corresponding tokens.

Reply:
Yup - that's the pain I'm talking about. Maintaining both. 🙃